src/frontend/src/flows/addDevice/manage/index.ts

```diff
@@ -0,0 +1,115 @@
+import { html, render } from "lit-html";
+import {
+  networkIcon,
+  securityKeyIcon,
+  warningIcon,
+} from "../../../components/icons";
+
+const pageContent = () => html`
+  <style>
+    .flowChoiceContainer {
+      display: flex;
+      gap: 1rem;
+      margin-top: 1rem;
+    }
+    .flowOption {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: center;
+      border: 1px solid gray;
+      border-radius: 4px;
+      width: 100%;
+      padding: 1rem;
+      font-family: "Montserrat", sans-serif;
+      font-size: 1.2rem;
+      margin-bottom: 2rem;
+    }
+    .flowOption:hover,
+    .flowOption:focus {
+      outline: none;
+      box-shadow: none;
+    }
+    .flowIcon {
+      height: 52px;
+    }
+    .flowOptionTitle {
+      font-weight: 500;
+      margin: 0.5rem;
```

Comment:
nit: there's a lot of `rem`s in this `<style>`; for margin especially might be interesting to use `em` instead so that the margins are scaled with the font size, if we ever decide to make it bigger or smaller.